src/main/java/core/basesyntax/RandomSize.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RandomSize {
+    public static final int LOWER_BOUND = 1;
+    public static final int UPPER_BOUND = 46340;
+
+    private Random random = new Random();
+
+    public int getRandomSize() {
+        int size = random.nextInt(UPPER_BOUND - LOWER_BOUND) + LOWER_BOUND;
+        return size;
+    }
+}

```

Comment:
The class `RandomSize` should not be a utility class with static methods according to the checklist. Consider refactoring your solution to avoid using static methods and create an instance of the class where you need to use its methods.